### PR TITLE
chore: project addition by SurajG20 #1649

### PR DESCRIPTION
**Pull request by SurajG20**
Added Projects

## Related Issue
#1649 

## Description

Added Project as previous one had some conflicts

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
add issue number. you've added PR number

**Comment by SurajG20** [comment]
> add issue number. you've added PR number

What's the difference, this is first addition of project ? i removed my previous pull request.

**Comment by priyankarpal** [comment]
> > add issue number. you've added PR number
> 
> What's the difference, this is first addition of project ? i removed my previous pull request.

before creating the PR you need to create an issue first

File: DB/projects.json [hunk 1]
```diff
@@ -3802,6 +3802,37 @@
...
+  "Social_media": {
+    "gitHub": "https://github.com/SurajG20",
+    "LinkedIn": "https://www.linkedin.com/in/suraj-goswami01",
+    "Instagram": "#",
+    "YouTube": "#",
+    "Twitter": "#"
+  },
+  "Projects": [
+    {
+      "link": "https://www.surajgoswami.me/",
```
Comment (priyankarpal) [inline DB/projects.json:3817]: add github repo link

File: DB/projects.json [hunk 2]
```diff
@@ -3802,6 +3802,37 @@
...
+      "title": "Portfolio Website",
+      "description": "My Personal Portfolio made in React and TailwindCss deployed on Vercel",
+      "tech": ["React JS", "Tailwind CSS"]
+    },
+    {
+      "link": "https://cinemato-two.vercel.app/",
```
Comment (priyankarpal) [inline DB/projects.json:3823]: add github repo link

File: DB/projects.json [hunk 3]
```diff
@@ -3802,6 +3802,37 @@
...
+      "title": "Movie App",
+      "description": "A Movie app made in React using TMDB api deployed on Vercel",
+      "tech": ["React JS", "Sass"]
+    },
+    {
+      "link": "https://surajg20.github.io/Weather-App/",
```
Comment (priyankarpal) [inline DB/projects.json:3829]: add github repo link

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 1 above)
Comment (SurajG20) [reply DB/projects.json]: just added can you check?

**Review by priyankarpal — APPROVED** [review]
Thank you for contributing your projects here 👏🏻 🎉

**Pull request merged**